src/models/UserModel.ts

```diff
@@ -28,6 +28,9 @@ export class UserModel {
   @Column()
   admin: boolean;
 
+  @Column({ default: true })
+  is_active: boolean;
```

Comment:
You should use camelCase here and for all other references to this variable:
```suggestion
  isActive: boolean;
```

I'm looking through the codebase and I see a mix of snake_case and camelCase, ideally we should standardize everything to camelCase, since this is the JS convention -- for now it's fine if you only modify isActive though

Reply:
Actually, another question I have about soft deletion is whether you guys have thought about what happens when attempting to fetch a user whose profile has been soft deleted? Ideally we should filter out all users who are not active in all the `getUser___()` Controllers (i.e., `getUsers`) which you can do via `.filter()` in JS, and throw errors when attempting to fetch the profile of a specific user who has been soft deleted (`getUserByGoogleId`, `getUserById`, `getUserByEmail`, etc)